### PR TITLE
Added appendix with DHS references and disclaimers

### PR DESCRIPTION
**Pull request by nissimsan**
Encouragement to consider if you should have your own contract numbers represented too. If so, please open issues to track for yourselves, or paste your contract numbers into the suggest changes feature of github.

### CODE REVIEW

**Comment by csuwildcat** [comment]
@nissimsan can you provide more info on why this docs addition is being requested? I'm unaware of DHS funding or contributing to any aspect of Sidetree, so I'm not sure if this statement applies.

**Comment by OR13** [comment]
@csuwildcat I requested this be added to all work that we (Transmute) have done on behalf of the contracts we have with DHS SVIP... this included work on sidetree, support for curves other than secp256k1, and ledgers other than bitcoin and ethereum.

**Comment by csuwildcat** [comment]
@OR13 I'm not sure this is something we'd put into a DIF repo, given anyone could be funding any DIF member external to DIF's awareness. Is there a motivation for noting a particular member's backing sponsor? Obviously there's some interesting dynamics with this particular sponsor, but I'd want to defer the inclusion of such assertions to Balazs and DIF ops folks to understand if external sponsorship-of-member notes are something to do in general.

**Comment by OR13** [comment]
@csuwildcat its pretty common for specs developed in SDO's to include acknowledgements:

https://www.w3.org/TR/vc-data-model/

> Portions of the work on this specification have been funded by the United States Department of Homeland Security's Science and Technology Directorate under contract HSHQDC-17-C-00019. The content of this specification does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred.

https://w3c.github.io/did-core/#acknowledgements

> Portions of the work on this specification have been funded by the United States Department of Homeland Security's (US DHS) Science and Technology Directorate under contracts HSHQDC-16-R00012-H-SB2016-1-002, and HSHQDC-17-C-00019, as well as the US DHS Silicon Valley Innovation Program under contracts 70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000030, 70RSAT20T00000045, 70RSAT20T00000003, and 70RSAT20T00000033. The content of this specification does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred.

That being said, if DIF is not prepared to include funding disclosures in specs like Sidetree, feel free to reject the PR.

We're adding these disclosures to our CCG / W3C work items, and we assumed that DIF would want them added here as well.

Also ping @troyronda @tplooker not sure if you have any work on Sidetree from a DHS perspective, but you might want to also include your contract numbers here as well.

GitHub will remember this thread, and we've already talked about our sidetree work pretty publicly, including here:

https://github.com/transmute-industries/sidetree.js

**Comment by csuwildcat** [comment]
For all the reasons @troyronda detailed, I am similarly uneasy and concerned about the refusal/reluctance to remove the At-Risk flag after we, in good faith, cleared the technical/implementation hurdles required (at least afaik). I would ask that the Chairs please remove this flag at their earliest opportunity.

**Review by OR13 — DISMISSED** [review]

**Comment by nembal** [comment]
I will circulate this question with DIF SC/TSC to formulate a shared opinion.

**Comment by OR13** [comment]
@thehenrytsai please consider merging this PR.

**Comment by csuwildcat** [comment]
I really don't feel comfortable with third-party funding attributions, even aside from the particular party that funded this. Why couldn't every startup member that worked on this have their VCs submitting attributions that say "Portions of the work on this specification have been funded by ABC Capital through investment in XYZ Startup". This just seems bizarre to me, so I am personally opposed to merging any attributions besides the names of the people/companies who actually did the work.

**Comment by OR13** [comment]
@rh7 I have contacted the SC regarding this matter, please reply in an official capacity to resolve this issue.

**Comment by nembal** [comment]
@csuwildcat 
The TSC discussed @OR13's ask at the 7th July meeting. The TSC approved it, so acknowledgments (if any) are a part of DIF-approved specifications. @csuwildcat if you have objections about this, it is a TSC topic.

Note publishing back then was not yet established for TSC, but today morning I retroactively updates the TSC notes. (sry about the delay) 

https://github.com/decentralized-identity/TSC/blob/main/agenda.md <- notes

**Comment by bumblefudge** [comment]
Apologies, it seems it was me who dropped this ball:

> next step: add to template? add to /org/ work item lifecycle document?

File: docs/spec/appendix.md [hunk 1]
```diff
@@ -74,4 +74,7 @@ The following operation inputs are in the form of Sidetree REST API Operations.
 
 ```json
 [[insert: www/vectors/resolution/longFormResponseDidDocument.json]]
-```
\ No newline at end of file
+```
+
+### Acknowledgements
+Portions of the work on this specification have been funded by the United States Department of Homeland Security's (US DHS) Silicon Valley Innovation Program under contracts 70RSAT20T00000003, and 70RSAT20T00000033. The content of this specification does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred.
```
Comment (csuwildcat) [inline docs/spec/appendix.md:80]: I would suggest changing the language to: "ENTITY_NAME_HERE received funding from the United States Department of Homeland Security's (US DHS) Silicon Valley Innovation Program to contribute to this work item under contracts 70RSAT20T00000003, and 70RSAT20T00000033. This work item does not necessarily reflect the position or the policy of the U.S. Government and no official endorsement should be inferred."

I would have made an actual code change suggestion, but I am not aware of what entity/entities they are, so if someone can add that in comments I can make an actionable code suggestion.

**Review by csuwildcat — CHANGES_REQUESTED** [review]

File: docs/spec/appendix.md (hunk 1 above)
Comment (OR13) [reply docs/spec/appendix.md:80]: I'm ok with the suggested changes, but I won't implement them unless a formal process defines this for all WGs.

Comment (OR13) [reply docs/spec/appendix.md:80]: @csuwildcat I suggest you implement those changes here: https://github.com/decentralized-identity/org/pull/9#issuecomment-1011317880

Comment (nissimsan) [reply docs/spec/appendix.md]: @csuwildcat, good suggestion - better this way.

**Comment by nissimsan** [comment]
@csuwildcat - checking in. I reworded last week based on your good suggestion. 
Would you guys mind expediting this, pls?

**Comment by bumblefudge** [comment]
LGTM as per yesterday's TSC meeting.

**Comment by csuwildcat** [comment]
Let's get it in line with the template @bumblefudge put together in the /org doc. The current text is missing his last bit about DIF not lending or implying support/affiliation with the outside entity as a part of the acknowledgement.

**Comment by nissimsan** [comment]
@csuwildcat, added.

**Comment by bumblefudge** [comment]
any update?

**Comment by nissimsan** [comment]
@csuwildcat, pls...?

**Comment by nissimsan** [comment]
@csuwildcat, it's 1.5 months ago I reacted on your change request. Would you mind unblocking, pls?

**Comment by bumblefudge** [comment]
@csuwildcat @troyronda @thehenrytsai anything stopping this from being merged at or before tomorrow's Sidetree WG meeting?

**Review by csuwildcat — APPROVED** [review]

**Pull request merged**